ext/data/schemas/dictionary-term-bank-v3-schema.json

```diff
@@ -51,6 +51,10 @@
                                 },
                                 "data": {
                                     "$ref": "#/definitions/structuredContentData"
+                                },
+                                "lang": {
+                                    "type": "string",
+                                    "description": "Defines the language of an element in the format defined by RFC 5646"
```

Comment:
Nitpick: add period to the end of description value (other descriptions are punctuated).

Apply to other locations in the file also.